fqlx/client.js

```diff
@@ -0,0 +1,19 @@
+const { Client } = require('fauna')
+
+const clients = {}
+
+function getFQLXClient({ secret, endpoint }) {
+  const cacheKey = `${secret}/${endpoint}`
+
+  if (!clients[cacheKey]) {
+    clients[cacheKey] = new Client({
+      secret,
+      endpoint: new URL(endpoint),
+    })
+  }
+
+  return clients[cacheKey]
```

Comment:
What is this cache for? The underlying http2 client for the fqlx is a singleton class, so multiple Clients will still use the same http2 instance. The http2 client keeps a map of open sessions keyed by endpoint. All that is to say, if you call `new Client()` with a given endpoint that has already been used, it will already reuse the existing session for that endpoint.

Reply:
This cache is copy/paste from the v4 client.js in this library. I can remove it